Solutions/CohesitySecurity/Playbooks/Incident_VM_Playbook/Incident_VM_Playbook.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
Please update this playbook from yings-instance. Have added more fields in HTTP request. It won't work otherwise.

Reply:
just update this.